packages/web/src/components/gcds-file-uploader/gcds-file-uploader.stories.tsx

```diff
@@ -62,48 +62,48 @@ export default {
   },
 };
 
-const Template = (args) => `
+const Template = (args) => (`
 // Web Component (Angular, Vue)
 <gcds-file-uploader
   uploader-id="${args.uploaderId}"
   label="${args.label}"
-  hint="${args.hint}"
-  error-message="${args.errorMessage}"
-  required="${args.required}"
-  disabled="${args.disabled}"
-  value="${args.value}"
-  accept="${args.accept}"
-  multiple="${args.multiple}"
-  lang="${args.lang}"
+  ${args.hint ? `hint="${args.hint}"` : null}
```

Comment:
Same question about the hint: is it set by default on purpose to encourage people to use it?